client/components/catalog/Card/index.vue

```diff
@@ -65,18 +84,29 @@ export default {
     description: ({ repository }) => repository.description,
     schema: ({ repository }) => getSchema(repository.schema).name,
     lastActivity: ({ repository }) => first(repository.revisions),
-    isPinned: ({ repository }) => get(repository, 'repositoryUser.pinned', false)
+    isPinned: ({ repository }) => get(repository, 'repositoryUser.pinned', false),
+    tags: ({ repository }) => repository.tags,
+    exceedTagLimit: ({ repository }) => repository.tags.length < TAG_LIMIT
   },
   methods: {
-    ...mapActions('repositories', ['pin']),
+    ...mapActions('repositories', ['pin', 'removeTag']),
     navigateTo(name = 'repository') {
       if (window.getSelection().toString()) return;
       this.$router.push({
         name,
         params: { repositoryId: this.repository.id }
       });
+    },
+    deleteTag(id, name) {
```

Comment:
Pass tag object (in order to do that don't destructure it in for). Rename to showDeleteConfirmation.